lib/android.js

```diff
@@ -29,13 +29,25 @@ function getApi () {
       },
       '_ZN3art6Thread14CurrentFromGdbEv': ['art::Thread::CurrentFromGdb', 'pointer', []],
       '_ZNK3art6Thread13DecodeJObjectEP8_jobject': ['art::Thread::DecodeJObject', 'pointer', ['pointer', 'pointer']],
-      '_ZN3art6mirror6Object5CloneEPNS_6ThreadE': ['art::mirror::Object::Clone', 'pointer', ['pointer', 'pointer']]
+      '_ZN3art6mirror6Object5CloneEPNS_6ThreadE': function (address) {
+        this["art::mirror::Object::Clone"] = new NativeFunction(address, 'pointer', ['pointer', 'pointer']);
+      },
+      '_ZN3art6mirror6Object5CloneEPNS_6ThreadEm': function (address) {
+        const nativeFn = new NativeFunction(address, 'pointer', ['pointer', 'pointer', 'int']);
+        this["art::mirror::Object::Clone"] = function (thisPtr, threadPtr) {
+          return nativeFn(thisptr, threadptr, 0);
```

Comment:
Thanks!

^ these two guys also need to be renamed.

One more thing, I noticed Xposed uses `size_t` for the last argument – we don't have a type for that yet in the `NativeFunction` API, so the most correct thing would be `pointer` as it's the same size (and use the `NULL` short-hand). That will look pretty weird though, so maybe we can make that clearer by doing:

```js
          const numTargetBytes = NULL;
          return nativeFn(thisptr, threadptr, numTargetBytes);
```